app/controllers/catalog_controller.rb

```diff
@@ -71,14 +71,16 @@ class CatalogController < ApplicationController
     # ------------------------------------------------------------------------------------------
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
-    config.add_index_field solr_name("repository",        :stored_sortable), label: "Library", helper_method: :render_repository_facet_link
-    config.add_index_field solr_name("unittitle",         :displayable),  :label => "Title", :highlight => true, :helper_method => :render_field_item
-    config.add_index_field solr_name("abstract",          :displayable),  :label => "Abstract", :highlight => true, :helper_method => :render_field_item
+    #config.add_index_field solr_name("unittitle",         :displayable),  :label => "Title", :highlight => true, :helper_method => :render_field_item
     config.add_index_field solr_name("format",            :displayable),  :label => "Format", :helper_method => :render_field_item
-    config.add_index_field solr_name("language",          :displayable),  :label => "Language", :helper_method => :render_field_item
-    config.add_index_field solr_name("unitdate",          :displayable),  :label => "Dates", :helper_method => :render_field_item
-    config.add_index_field solr_name("collection",        :displayable),  :label => "Archival Collection", :helper_method => :render_collection_facet_link, :highlight => true
-    config.add_index_field solr_name("parent_unittitles", :displayable),  :label => "Series", :highlight => true, :helper_method => :render_series_facet_link
```

Comment:
@barnabyalter ,

Yes, I thought about this. I kept going back and forth and would have really liked to use existing functionality. The reason why I didn't do that is because it doesn't replicate it exactly. I tried using render_series_facet_link but I think the way the series_sortable and series_facetable functions work(with the facetable and sortable params), it doesn't allow me to grab the object's unit title if I am at the object or item level. I didn't want to futz to much around the series sortable and facetable functions since they were also being used for something else. I can try again to see if I can get it to work...